readthedocs.yml

```
@@ -1,8 +1,12 @@
 version: 2
 sphinx:
   configuration: docs/source/conf.py
+conda:
+  file: docs/environment.yml
+  environment: jupyterlab_documentation
```

from [rtd docs](https://docs.readthedocs.io/en/stable/config-file/v2.html#conda), I think it should look like this:

```suggestion
conda:
  environment: docs/environment.yml
```